Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -870,4 +912,79 @@ void ABenchmarkGymGameModeBase::GenerateTotalNumsForActors(const FString& Worker
 			}
 		}
 	}
+}
+
+void ABenchmarkGymGameModeBase::GetVelocityForMovementReport()
+{
+	// Report logic
+	if (RequiredPlayerMovementReportTimer.HasTimerGoneOff())
+	{
+		FVector2D AvgVelocity = FVector2D(0.0f, 0.000001f);
+		// Loop each players
+		GetPlayersVelocitySum(AvgVelocity);
```

Comment:
Concerned that it might be easy to ignore the a single player not moving

Reply:
Actually we use the latest 10 times average velocity to judge whether the players stop moving, we not use one time's velocity for expectation. Also we calculate the total velocity of all players and get the average, only if the average is smaller than our expectation( current our expectation is 1, and we have tested for 100 players, the average velocity is between 9.7 to 11.4), it will failed.